x-pack/plugin/build.gradle

```diff
@@ -99,6 +99,9 @@ tasks.named("transformV7RestTests").configure({ task ->
   task.replaceKeyInDo("license.post_start_trial", "xpack-license.post_start_trial")
   task.addAllowedWarningRegex(".*_xpack/license.* is deprecated.*")
 
+  task.replaceKeyInDo("migration.deprecations", "xpack-migration.deprecations")
+  task.addAllowedWarningRegex(".*_xpack/migration.* is deprecated.*")
```

Comment:
maybe we could apply to the test that is using it?

Reply:
With #75950, there's only one test that's using the change (`"Test get SSL certificates"`) so it's straightforward to limit ourselves to just the one test. Here, though, there's two (`"Test Deprecations"` and `"Test ml"`) so I don't think it's worth the additional lines of code to limit ourselves.

I'm going to merge this as is, but if you disagree strongly we can always tweak change it as a new PR against `master`.